consensus/src/consensus.rs

```diff
@@ -242,7 +245,10 @@ impl<R: ConsensusRpc> ConsensusClient<R> {
         for update in updates {
             self.verify_update(&update)?;
             self.apply_update(&update);
-            self.store_finalized_update(&update, false)?;
+            if update.finalized_header.slot > base_slot {
+                self.store_updates_until_finality_update(&update.into())
+                    .await?;
+            }
```

Comment:
- MMR should be stored if `update.finalized_header.slot >= base_slot`.
- If verifier stores the finalized updates between `initial_checkpoint` to `stored_tip_slot`, it could be updated faster when restarts.

Reply:
in `bootstrap()`, the base slot has already been stored in storage, so it doesn't matter to care about the `update.finalized_header.slot == base_slot` case.